ansible_collections/arista/avd/roles/anta_runner/tasks/main.yml

```
@@ -50,7 +50,10 @@
       md_output: "{{ anta_report_md_path }}"
       json_output: "{{ anta_report_json_path }}"
       filters:
-        hide_statuses: "{{ anta_report_hide_statuses | arista.avd.default([]) }}"
+        exclude_statuses: "{{ anta_report_exclude_statuses | arista.avd.default([]) }}"
+      sorting:
+        sort_by: "{{ anta_report_sort_by | arista.avd.default([]) }}"
+        status_priority: "{{ anta_report_status_priority | arista.avd.default([]) }}"
```

We have default values for these.
```suggestion
        sort_by: "{{ anta_report_sort_by }}"
        status_priority: "{{ anta_report_status_priority }}"
```

---

Updated.